api/rest/restcore/issues_rest.php

```diff
@@ -368,6 +368,9 @@ function rest_issue_update( \Slim\Http\Request $p_request, \Slim\Http\Response $
 
 	# Construct full issue from issue from db + patched info
 	$t_issue_patch = $p_request->getParsedBody();
+	if( !$t_issue_patch ) {
+		return $p_response->withStatus( HTTP_STATUS_BAD_REQUEST, "Empty Body");
```

Comment:
There are several reasons why the getParsedBody() would fail:

1. Content-Type header is not passed, rather than passed to indicate json.
2. The body is set to '{}'
3. The body is set to ''
4. The body is set to invalid json '{'

So empty body may not be the right error to return here, but `Invalid request body or format`.

Reply:
Fixed